### DISCUSSION TITLE
How can I use the ImageSelector?

### DISCUSSION THREAD

**Discussion by BEDIRHANSAMSA** [HandyOrg/HandyControl#992] [Q&A] [answered]
Hi, How can I trigger the method after the user selects an image? Thanks.

**Comment by ghost1372** [comment, marked as answer]
There is currently no way to do that
But I created an event for this purpose and I made a [pull request](https://github.com/HandyOrg/HandyControl/pull/1001) for it.
also i converted this Q&A to an [Issue](https://github.com/HandyOrg/HandyControl/issues/1000) so you can track it

**Reply by BEDIRHANSAMSA** [reply]
Thank you so much.
